workflows/ascc_genomic.nf

```diff
@@ -111,13 +87,14 @@ workflow ASCC_GENOMIC {
     } else {
         log.warn("MAKE SURE YOU ARE AWARE YOU ARE SKIPPING ESSENTIAL JOBS, THIS INCLUDES BREAKING SCAFFOLDS OVER 1.9GB, FILTERING N\'s AND GC CONTENT REPORT (THIS WILL BREAK OTHER PROCESSES AND SHOULD ONLY BE RUN WITH `--include essentials`)")
 
-        reference_tuple_from_GG = ch_samplesheet // This is the reference genome input channel
+        reference_tuple_from_GG = ch_samplesheet
+        ej_dot_genome           = Channel.of([[],[]])
+        ej_gc_coverage          = Channel.of([[],[]])
+        reference_tuple_w_seqkt = Channel.of([[],[]])
     }
 
 
-    if ( (include_workflow_steps.contains('kmers') || include_workflow_steps.contains('ALL')) &&
-            !exclude_workflow_steps.contains("kmers")
-    ) {
+    if ( params.kmers.contains("both") || params.kmers.contains("genomic") ) {
```

Comment:
See `==` comment above, and for all other cases

Reply:
Done